appveyor.yml

```diff
@@ -3,6 +3,9 @@ build: off
 cache:
   - 'C:\\tmp'
 environment:
+  global:
+    # Please, change to use token of PyCQA/pylint
+    COVERALLS_REPO_TOKEN: "R8noz2lfRpFXNPKQirwNHJyLl1wMIJnYF"
```

Comment:
What?! No, tokens shouldn't be shared and definitely they shouldn't be inside a configuration file. Why do you need it in the first place?